src/ls/ls.rs

```diff
@@ -191,9 +192,10 @@ fn sort_entries(entries: &mut Vec<PathBuf>, options: &getopts::Matches) {
             entries.sort_by_key(|k| get_metadata(k, options).map(|md| md.ctime()).unwrap_or(0));
```

Comment:
Oh wait, I may have merged this too quickly.  Does this need to be reversed as well?